libs/design/src/molecules/callout/callout.component.scss

```diff
@@ -0,0 +1,152 @@
+@import '../../scss/daff-util';
+
+@mixin callout-headings($titleSize: 2.5rem, $bodySize: 1rem) {
```

Comment:
What's the point of this mixin?

Reply:
Not sure, I replicated what we did on Graycore. I don't remember if you had a reasoning for this.

Reply:
If it's not used more than once, remove and merge into the class.